Lib/test/test_pathlib.py

```diff
@@ -1713,21 +1713,6 @@ def test_resolve_dot(self):
         # Non-strict
         self.assertEqual(r.resolve(strict=False), p / '3' / '4')
 
-    def test_with(self):
```

Comment:
There should still be a test that `with Path` works, just that it's a no-op.

Reply:
I wasn't sure how to do that - what's the opposite of `assertRaises()`?

Reply:
Nothing, just regular code that doesn't raise :-)